zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -150,7 +152,7 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
   };
 
   $scope.killSaveTimer = function() {
```

Comment:
I've seen a lot of other function/variables (like renderAngular in paragraph.controller.js) that should be removed from $scope, and plan to do that in a separate clean up PR.